OpenShift/nexus-centos.json

```diff
@@ -216,30 +202,30 @@
                               "readinessProbe": {
                                  "httpGet": {
                                     "path": "/service/metrics/ping",
-                                    "port": "web-port",
+                                    "port": 8081,
```

Comment:
I'm going to change this back and test... initially this change was made because liveness probe was failing.  I've determined that it just needed more time to start before the probe fired off.